workbench-core/swb-common-ui/src/api/users.ts

```diff
@@ -16,7 +16,8 @@ const createUser = async (createUserForm: CreateUserForm): Promise<void> => {
   await httpApiPost(`users`, { ...createUserForm });
 };
 
-const useUsers = () => {
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const useUsers = (): any => {
```

Comment:
do we need to specify any? as we are explicitly giving the return object

Reply:
Unfortunately yes we do. The way files are compiled/linted here is different than `solutions/swb-ui`, which requires us to specify any (type of `users`)